runtime/browser/xenon-base.js

```diff
@@ -0,0 +1,80 @@
+let XenonStaterMixin = Base => class extends Base {
```

Comment:
Some of it we'll need quite soon, some of it we can remove, I should be able to tighten it up soon.